### ISSUE TITLE
[Bug] Await missing in createEmptySegmentationForViewport

### ISSUE THREAD

**Issue by Billuc** [OHIF/Viewers#4207]
### Describe the Bug

I am writing a function that creates a new segmentation and then performs actions on the newly created segmentation. I am calling the `createEmptySegmentationForViewport` command to create a new segmentation. However, when the function ends, I don't see any new segmentation:

```ts
// CREATE NEW (EMPTY) SEGMENTATION
const command = commandsManager.getCommand(
	"createEmptySegmentationForViewport",
	"SEGMENTATION",
);
await command.commandFn({ viewportId: viewportGridService.getActiveViewportId() });

console.log({...segmentationService.segmentations}); // The new segmentation isn't there
```

but the segmentation does appear in the UI. It appears that an async function in the `createEmptySegmentationForViewport` isn't awaited:

```ts
[...]
updateViewportsForSegmentationRendering({...});
// instead of await updateViewportsForSegmentationRendering({...});
```

Is this on purpose or is it a bug ? 

### Steps to Reproduce

1. Call the createEmptySegmentationForViewport command
2. Get the segmentations via the segmentationManager right after

### The current behavior

The new segmentation isn't there because it is added in an async function that is not awaited

### The expected behavior

After the call to the command, the newly created segmentation appears in the segmentationService

### OS

Fedora 40 with Docker (Debian 12 Bookworm)

### Node version

18.20.3

### Browser

Chromium

**Comment by sedghi** [comment]
Yeah we should await it i guess

**Comment by sedghi** [comment]
FYI: After recent changes in OHIF 3.9 to `createLabelmapForViewport`, since our default creation is now stack, awaiting volume creation is unnecessary.

**Comment by sedghi** [comment]
We released [OHIF 3.9](https://github.com/OHIF/Viewers/tree/release/3.9) which address this issue
